cli/js/ops/fs/truncate.ts

```diff
@@ -1,30 +1,31 @@
 // Copyright 2018-2020 the Deno authors. All rights reserved. MIT license.
+
 import { sendSync, sendAsync } from "../dispatch_json.ts";
 
 function coerceLen(len?: number): number {
-  if (!len) {
-    return 0;
-  }
-
-  if (len < 0) {
+  if (!len || len < 0) {
     return 0;
   }
 
   return len;
 }
 
 export function ftruncateSync(rid: number, len?: number): void {
-  sendSync("op_ftruncate", { rid, len: coerceLen(len) });
+  len = coerceLen(len);
```

Comment:
Personal opinion, this actually add more code, and we should try to avoid reassignment of arguments.